src/components/reminder-form/reminder-form.tsx

```diff
@@ -1,18 +1,46 @@
-import React, { FormEvent } from "react";
+import React, { FormEvent, useState } from "react";
 
-const Form = ({}: {}) => {
+
+const Form = ({ }: {}) => {
+  const [taskName, setTaskName] = useState('');
+  const [recurring, setRecurring] = useState(false);
   const handleSubmit = (e: FormEvent<HTMLFormElement>): void => {
     e.preventDefault();
   };
   return (
     <form onSubmit={handleSubmit} className="form-group">
       <div>
-        <input type="text" className="form-control" placeholder="Add a remider" />
+        <input type="text" className="form-control" placeholder="Remider Title" />
+      </div>
+      <div className="card mt-3 text-secondary ">
+        <h6>What would you like the reminder to say?</h6>
+        <textarea id="custom-message" ></textarea>
+      </div>
+      <div className="mt-3">
+        <input type="text" className="form-control" placeholder="How would you like to be reminded?" />
+      </div>
+      <div className=" card mt-3 text-secondary">
+        <h6>When would you like to be reminded?</h6>
+        <input type="date"></input>
+      </div>
+      <div className="card mt-3 text-secondary" >
+        <h6>Is this a recurring task?</h6>
+        <button className="btn btn-primary col-2" type="button"> YES</button>
+      </div>
+      <div className="card mt-3 text-secondary">
+        <h6>How often would would like to be reminded?</h6>
+        <select id="reminder-frequency" name="reminder">
+          <option value="hourly">Hourly</option>
```

Comment:
You can use enum values here, instead of hard coding the values inside options

Reply:
How?

Reply:
I tried it but I e.target.value doesn't work What can I use instead of value?

Reply:
I will file a PR with this change to show how I would do it. One day :sweat_smile: